lib/astarte/utilities/params_gen.ex

```diff
@@ -33,18 +33,20 @@ defmodule Astarte.Generators.Utilities.ParamsGen do
   - **Seamless Integration:** Built on top of ExUnitProperties, it integrates smoothly with your property-based tests.
   - **Flexible Customization:** Accepts a keyword list for generator overrides, ensuring that only the generators you specify are replaced while the rest remain untouched.
   - **Improved Test Clarity:** By explicitly defining custom generators, tests become easier to understand and maintain.
+  - **Labeled Clauses:** Bind overrides to a specific clause using labels to avoid conflicts and improve readability.
+  - **Destructuring Requires Label:** Supports the same destructuring on the left-hand side as `gen all` (e.g., `%{k: v} = var <- ...`), but only when the clause is labeled. Without a label, destructuring is not allowed.
```

Comment:
I find this ambiguous, as I don't immediately understand if you mean the left side of the `<-` or the `=`. I suggest removing the assignment to remove the ambiguity 


```suggestion
  - **Destructuring Requires Label:** Supports the same destructuring on the left-hand side as `gen all` (e.g., `%{k: v} <- ...`), but only when the clause is labeled. Without a label, destructuring is not allowed.
```